src/cloudkit.cs

```diff
@@ -960,11 +959,7 @@ interface CKNotification : NSSecureCoding {
 
 		[Static]
 		[Export ("notificationFromRemoteNotificationDictionary:")]
-#if XAMCORE_4_0
```

Comment:
Cause I added it due to changes in the API that typed the dictionary but Apple reverted that change. Ergo, I remove it to make sure that we do not change the API.